WireAuthentication/Sources/WireAuthenticationUI/Views/Login/LoginViaEmail/LoginViaEmailViewModel.swift

```diff
@@ -93,7 +93,7 @@ package final class LoginViaEmailViewModel: ObservableObject {
         environmentType != .production
     }
 
-    func submitPassword(_ password: String) async {
+    func submit(password: String, proxyEmail: String?, proxyPassword: String?) async {
```

Comment:
suggestion: instead of passing two optionals, pass a single optional tuple: `proxyCredentials: (email: String, password: String)?`

Reply:
Done